client/client_update_test.go

```diff
@@ -1087,6 +1099,14 @@ func TestUpdateNonRootRemoteCorruptedCannotUseLocalCache(t *testing.T) {
 	if testing.Short() {
 		t.Skip("skipping test in short mode")
 	}
+
+	acceptDelgAnyway := map[string]struct{}{
```

Comment:
I think it might be worth factoring this out into a global var like `waysToMessUpServer`